hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/compactions/CloseChecker.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hbase.regionserver.compactions;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.regionserver.Store;
+import org.apache.yetus.audience.InterfaceAudience;
+
+/**
+ * Check periodically to see if a system stop is requested
+ */
+@InterfaceAudience.Private
+public class CloseChecker {
```

Comment:
Does it have to be public? Can it be package local? It is only used in this package?

Reply:
I think it would be good to use `package local`.
However, The package of `FaultyMobStoreCompactor` is not the same as `CloseChecker`.
The package of `FaultyMobStoreCompactor` is `org.apache.hadoop.hbase.mob`.

Reply:
dang.

ok. public for now.